src/app/content/components/Section.tsx

```diff
@@ -0,0 +1,110 @@
+import { Box, Heading } from '@chakra-ui/layout';
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Badge,
+  Flex,
+  Popover,
+  PopoverArrow,
+  PopoverBody,
+  PopoverCloseButton,
+  PopoverContent,
+  PopoverTrigger,
+} from '@chakra-ui/react';
+import React from 'react';
+import { MeetingTimes, Seat } from '../../../fetchers';
+import { Schedule } from './Schedule';
+import { SeatInfo } from './Seats';
+
+export interface SectionInfoProps {
+  crn: string;
+  section: string;
+  sectionCode: string;
+  instructionalMethod: string;
+  additionalNotes?: string;
+  meetingTimes: MeetingTimes[];
+  seat?: Seat;
+}
+
+export function SectionInfo({
+  crn,
+  sectionCode,
+  instructionalMethod,
+  additionalNotes,
+  seat,
+  meetingTimes,
+}: SectionInfoProps): JSX.Element {
+  const isASYNC = additionalNotes?.indexOf('asynchronous') !== -1;
+  const isSENG = additionalNotes?.indexOf('Reserved for BSENG students') !== -1;
+  const isCSC = additionalNotes?.indexOf('Reserved for students in a Computer Science program') !== -1;
+
```

Comment:
would we need more consts like these for other majors as well?

Reply:
Yup. These are pretty primitive checks so they need to be extended but it's a start.